### PR TITLE
Green check marks

### PR DESCRIPTION
**Pull request by dickmao**
We should strive for green check marks.

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -1,52 +1,38 @@
...
+    - '**.rst'
+    branches-ignore:
+    - 'master'
+    - 'main'
 
 jobs:
   build:
     runs-on: ubuntu-latest
     strategy:
-      fail-fast: false
```
Comment (conao3) [inline .github/workflows/test.yml:10]: Please keep this fail-fast config? Please tell me a reason to remove this?

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -1,52 +1,38 @@
...
-          - '26.2'
-          - '26.3'
-          - '27.1'
-          - 'snapshot'
-        include:
-          - emacs_version: 'snapshot'
-            allow_failure: true
+        os: [ubuntu-latest]
+        python_version: [3.6]
+        emacs_version: [24.4, 24.5, 25.3, 26.3, 27.1]
```
Comment (conao3) [inline .github/workflows/test.yml:23]: Please check every Emacs-24.1~Emacs27.1 and snapshot.
If you want to remove this (I think so too), we should finish the #523 discussion.

File: .github/workflows/test.yml (hunk 1 above)
Comment (dickmao) [reply .github/workflows/test.yml:10]: Restored 37052df

File: .github/workflows/test.yml (hunk 2 above)
Comment (dickmao) [reply .github/workflows/test.yml]: Addressed 37052df

**Comment by conao3** [comment]
It doesn't make much sense to keep it like this, so I'll merge it.
Well, we haven't been able to test Emacs-24.1 and 24.2, but I feel it's fair to support only Emacs-24.5 and above.
Anyway, it seems that cask can be tested more robustly with your contribution. Thanks!

**Pull request merged**